beacon_node/beacon_chain/src/execution_payload.rs

```diff
@@ -69,7 +70,7 @@ pub fn notify_new_payload<T: BeaconChainTypes>(
                 // This block has not yet been applied to fork choice, so the latest block that was
                 // imported to fork choice was the parent.
                 let latest_root = block.parent_root();
-                chain.process_invalid_execution_payload(
+                chain.clone().process_invalid_execution_payload(
```

Comment:
Is this clone necessary? It's compiling for me without it

Reply:
Thanks, added these before I realised I should use `self: &Arc<Self>`